radish34/README.md

```diff
@@ -166,9 +166,9 @@ After you've done that, log in to the Github package registry by running
 7. Run `docker-compose up -d && docker-compose restart`
    - This will reinstate and restart all `radish` containers
    - Wait about 10 seconds to give containers time to complete their initialization routines
-   - Run `docker-compose logs -f {SERVICE_NAME}` to check the logs of specific containers. Key ones to watch are: `api-{role}` and `ui`
+   - Run `docker-compose logs -f {SERVICE_NAME}` to check the logs of specific containers. Key ones to watch are api-{role} and ui. For example: `docker-compose logs -f api-buyer docker-compose logs -f api-supplier1 docker-compose logs -f api-supplier2 docker-compose logs -f ui`
```

Comment:
Could you change the example command on this line to `docker-compose logs -f api-buyer api-supplier1 api-supplier2 ui`

Reply:
@breakpointer  - I added it now.